tests/plotting/test_collection.py

```diff
@@ -0,0 +1,15 @@
+"""This module contains any tests which cause memory leaks."""
+import numpy as np
+
+
+def test_pyvistandarray_assign(sphere):
+    sphere.point_data['data'] = range(sphere.n_points)
+
+    # this might leave a reference behind if we don't property use the pointer
+    # to the vtk array.
+    sphere.point_data['data'] = sphere.point_data['data']
+
+
+def test_pyvistandarray_strides(sphere):
+    sphere['test_scalars'] = sphere.points[:, 2]
+    assert np.allclose(sphere['test_scalars'], sphere.points[:, 2])
```

Comment:
I assume this file is meant as a regression test for now-fixed VTK leaks. Is there any way we can verify that no leaks are happening? Or is this already subject to the usual VTK GC checking? If so, did these tests fail before this fix?

Reply:
All files in the `tests/plotting` directory are subject to our VTK GC checking. These tests failed before this.